crates/xxi-node/src/node/dlc_channel.rs

```diff
@@ -302,8 +315,18 @@ impl<D: BdkStorage, S: TenTenOneStorage + 'static, N: LnDlcStorage + Sync + Send
                     Some(protocol_id),
                 )?;
 
+                // TODO(holzeis): Separate into different functions for renew and rollover.
```

Comment:
🔧 I think we should do this straight away :D

Also, if we want to use 10101 terminology consistently, I think we should not use the word `renew`. The DLC channel renew protocol corresponds to open-position-after-settle, resize-position or rollover. I think we should stick with those at this level. Since the first two are interchangeable we could say `propose_reopen_or_resize`?